_posts/2018-05-16-cockpit-168.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Cockpit 168
+author: pitti
+date: 2018-05-16 11:00
+tags: cockpit linux
+slug: cockpit-168
+category: release
+summary: Cockpit with improved root privilege check
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly.
+Here are the release notes from version 168.
+
+### Improve checks for root privilege availability
+
+Many actions, like rebooting the machine or joining an IPA domain, can only be
+done by an administrator (`root`). To decide whether to enable these actions,
+Cockpit previously checked if the user was a member of the `wheel`
+(RHEL/Fedora) or `sudo` (Debian) groups.
+
+This was insufficient: Other operating systems or local customizations might
+use other groups, or have user instead of group based sudo rules. For example
+with FreeIPA central user management, sudo access is given through the
+`admins@domain.name` group.
+
+Cockpit now does not make any assumptions about special groups any more, and
+instead just checks if the user is capable of running commands as `root`. Most
+notably this makes privileged operations work out of the box with FreeIPA.
```

Comment:
Cockpit no longer makes assumptions about special groups. Now, Cockpit simply
checks if the user is capable of running commands as `root`.

As a result of this fix, privileged operations now work by default with
[FreeIPA](https://freeipa.org/), as it uses an `admins@domain.name` group.